src/main/java/com/lucasdavid/basicAuthAPI/entities/User.java

```diff
@@ -0,0 +1,62 @@
+package com.lucasdavid.basicAuthAPI.entities;
+
+import jakarta.persistence.*;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Table(name = "tb_users")
+public final class User implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String username;
+    private String password;
```

Comment:
It's usually a good a idea to add a salt the password:

Password salting is a security technique that enhances password protection by adding a random string (the "salt") to a password before hashing it. This makes it significantly harder for attackers to crack passwords, even if they have access to a database of hashed password